qiskit/extensions/standard/t.py

```diff
@@ -78,20 +89,35 @@ def inverse(self):
         return TGate()
 
     def to_matrix(self):
-        """Return a Numpy.array for the S gate."""
+        """Return a numpy.array for the inverse T gate."""
         return numpy.array([[1, 0],
-                            [0, (1-1j) / numpy.sqrt(2)]], dtype=complex)
+                            [0, (1 - 1j) / numpy.sqrt(2)]], dtype=complex)
+
+
+class TdgGate(TInvGate, metaclass=TInvMeta):
+    """The deprecated TInv gate."""
+
+    def __init__(self):
+        warnings.warn('TdgGate is deprecated, use TInvGate instead!', DeprecationWarning, 2)
```

Comment:
Can the deprecation warnings be updated to include the information from https://qiskit.org/documentation/contributing_to_qiskit.html#deprecation-warnings ?

Reply:
Yes, what version should I indicate? As in  ```... will be deprecated as of 0.x.x```.

Reply:
Terra's next minor release is 0.12.0, so we can aim for that.